docs/rules/no-unguarded-risky-node-api-in-ssrable-components.md

```diff
@@ -0,0 +1,29 @@
+# Disallow unguarded risky Node API calls in SSR context (`lwc/no-unguarded-risky-node-api-in-ssrable-components`)
+
+The rule disallows the use of unguarded risky Node API calls within the context of server-side rendering. These calls can lead to security vulnerabilities and unexpected behavior when rendering components on the server. Instead, any risky operations should be properly guarded to ensure they are only executed in safe contexts.
```

Comment:
We want the linter to error if the Node APIs are used, even if they are guarded in some way.

```suggestion
The rule disallows the use of unsupported Node API calls. These calls can lead to security vulnerabilities and unexpected behavior when rendering components on the server.
```